src/tapis-app/Dashboard/_components/Dashboard/Dashboard.tsx

```diff
@@ -107,6 +109,14 @@ const Dashboard: React.FC = () => {
               counter={`${jobs?.data?.result?.length} jobs`}
               loading={jobs?.isLoading}
             />
+            <DashboardCard
+              icon="project"
+              name="Projects"
```

Comment:
Can you rename this to Streams? Since that is the top level name of the service.